tensorflow/lite/kernels/test_util.h

```diff
@@ -15,6 +15,8 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_KERNELS_TEST_UTIL_H_
 #define TENSORFLOW_LITE_KERNELS_TEST_UTIL_H_
 
+#include <gmock/gmock.h>
```

Comment:
Please move these lines back to where they were.

Reply:
ok changed to original format.
Don't know why clang format make these changes.